app/views/admin/AdministerUserModal.js

```diff
@@ -287,6 +287,10 @@ module.exports = (AdministerUserModal = (function () {
       if (attrs.userTimeZone?.[0] === 'on') {
         timeZone = this.userTimeZone
       }
+      if (attrs.revokable?.[0] !== 'on') {
+        attrs.irrevocable = true
+      }
+      delete attrs.revokable
```

Comment:
why are we deleting after setting?

Reply:
oh since i use `irrevocable` in db (so that we don't need migrate old licenses) . and actually I think irrevocable is really rare, so do not add any key by default. and the delete just remove the unmatched key.

Reply:
wow, its irrevokable and revokable